lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -88,18 +87,30 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
     return element;
   }
 
+  /**
+   * @param {!ReportRenderer.AuditJSON} audit
+   * @return {number|undefined}
+   */
+  _getWastedMs(audit) {
+    try {
+      return audit.result.details.summary.wastedMs;
+    } catch (e) {
+      return -1;
```

Comment:
should this return undefined? it doesn't seem like the undefined case is really handled well now though, this is only for errors anyhoo right? maybe it always returns `wastedMs` or `MIN_VALUE` to force it to bottom or something?

Reply:
Yeah this is all about oppos that have errored.

I don't return undefined because of:

https://github.com/GoogleChrome/lighthouse/blob/cc5434b8f4a83fe37d1a9cb7841829031f113dc1/lighthouse-core/report/html/renderer/performance-category-renderer.js#L153-L154
![image](https://user-images.githubusercontent.com/39191/39682969-ac31d252-5167-11e8-9f68-5b1f099ad516.png)

min_value wfm!